app/test_split_integer.py

```diff
@@ -2,20 +2,26 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    result = split_integer(8, 4)
+    assert sum(result) == 8, "Sum of 'parts' should be equal to value"
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    result = split_integer(15, 3)
+    assert result == [5, 5, 5], ("Parts should equal to each other"
```

Comment:
remove all comments, the name of the function will give an understanding of what is being tested here